test/user-profile/reducers/profile.unit.spec.js

```diff
@@ -0,0 +1,22 @@
+import { expect } from 'chai';
+
+import reducer from '../../../src/js/user-profile/reducers/profile.js';
+import { UPDATE_LOGGEDIN_STATUS } from '../../../src/js/login/actions';
+import { PROFILE_LOADING_FINISHED } from '../../../src/js/user-profile/actions';
+
+describe('Profile reducer', () => {
```

Comment:
Huh, I guess there _wasn't_ a reducer unit test for user-profile. Should we add the other action (`UPDATE_PROFILE_FIELD`) in here too even though it isn't in the scope of this PR?